src/Traces.Common/Constants/TextConstants.cs

```diff
@@ -63,5 +63,8 @@ public static class TextConstants
         public const string ApaleoSetupButtonNavigateToApaleoText = "Go to apaleo app";
 
         public const string ApaleoSetupButtonTryAgainText = "Try again";
+
+        public const string RequestPropertyIdFromApaleoErrorMessage =
```

Comment:
```suggestion
        public const string RequestPropertyIdForReservationFromApaleoErrorMessage =
```

or something like that - because the way it's written now it sounds like you'd be checking if the property exists. Or you can call it some more generic way - fetching data from apaleo for traces stuff error thingie?